[2023-12-12T01:14:41.440+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_measures_data scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-12T01:14:41.455+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_openaq_catchup.load_measures_data scheduled__2023-12-11T00:00:00+00:00 [queued]>
[2023-12-12T01:14:41.455+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 4
[2023-12-12T01:14:41.496+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_measures_data> on 2023-12-11 00:00:00+00:00
[2023-12-12T01:14:41.505+0000] {standard_task_runner.py:57} INFO - Started process 329 to run task
[2023-12-12T01:14:41.512+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'dag_openaq_catchup', 'load_measures_data', 'scheduled__2023-12-11T00:00:00+00:00', '--job-id', '328', '--raw', '--subdir', 'DAGS_FOLDER/dag_openaq_catchup.py', '--cfg-path', '/tmp/tmp1u0nd34w']
[2023-12-12T01:14:41.522+0000] {standard_task_runner.py:85} INFO - Job 328: Subtask load_measures_data
[2023-12-12T01:14:41.705+0000] {task_command.py:416} INFO - Running <TaskInstance: dag_openaq_catchup.load_measures_data scheduled__2023-12-11T00:00:00+00:00 [running]> on host 1a187d37aaef
[2023-12-12T01:14:42.090+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_openaq_catchup' AIRFLOW_CTX_TASK_ID='load_measures_data' AIRFLOW_CTX_EXECUTION_DATE='2023-12-11T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-12-11T00:00:00+00:00'
[2023-12-12T01:14:42.100+0000] {main.py:140} INFO - Archivo de configuracion leido correctamente.
[2023-12-12T01:14:42.101+0000] {utils.py:25} INFO - Obteniendo datos de https://api.openaq.org/v2/measurements...
[2023-12-12T01:14:43.932+0000] {utils.py:73} INFO - Conexi√≥n a la base de datos establecida exitosamente.
[2023-12-12T01:14:46.617+0000] {utils.py:90} INFO - Cargando datos en la tabla stg_measurements...
[2023-12-12T01:14:48.666+0000] {utils.py:98} INFO - Datos cargados exitosamente en la tabla stg_measurements.
[2023-12-12T01:14:48.672+0000] {main.py:166} INFO - Ejecutando query CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();...
[2023-12-12T01:14:49.641+0000] {main.py:168} INFO - Se ejecuto correctamente la query: CALL nicolasmvinciguerra_coderhouse.petl_load_fact_measure();
[2023-12-12T01:14:49.973+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-12T01:14:50.025+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag_openaq_catchup, task_id=load_measures_data, execution_date=20231211T000000, start_date=20231212T011441, end_date=20231212T011450
[2023-12-12T01:14:50.119+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-12T01:14:50.202+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
